plugins/janus_recordplay.c

```diff
@@ -2073,8 +2130,7 @@ void janus_recordplay_update_recordings_list(void) {
 				*ext = '\0';
 			const char *textcodec = janus_recordplay_parse_codec(recordings_path,
 				rec->drc_file, NULL, sizeof(NULL), NULL);
-			if(textcodec)
-				rec->textdata = TRUE; /* Binary recordings not supported yet */
+			rec->textdata = textcodec && (!strcasecmp(textcodec, "text"));
```

Comment:
nit: no need for the extra parentheses around `!strcasecmp()` here

Reply:
True, I just thought it made it less ambiguous.